test/views/declaration/DucrEntryViewSpec.scala

```diff
@@ -127,6 +128,13 @@ class DucrEntryViewSpec extends PageWithButtonsSpec with Injector {
       }
     }
 
+    onJourney(STANDARD, CLEARANCE, OCCASIONAL, SIMPLIFIED)(aDeclaration(withStatus(DeclarationStatus.AMENDMENT_DRAFT))) { implicit request =>
+      "hide 'Back' button that links to 'Ducr Choice' page" in {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft